auth.py

```diff
@@ -1,32 +1,43 @@
-"""
-Flask Auth implementation
-"""
+from flask_login import login_required, LoginManager
+from flask_login import current_user
 
-from flask import g
-from flask_httpauth import HTTPBasicAuth
 from models import User
 
-auth = HTTPBasicAuth()
 
+login_manager = LoginManager()
 
-@auth.verify_password
-def verify(email, password):
-    """
-    Verify the request to api users endpoints, trying to get the user with the
-    provided email and verifying the password against the stored hashed one.
 
-    If the user is verified it will be stored inside `Flask.g` to be used from
-    the enpoint handler if needed, for example to allow `delete` or `put` only
-    on the user's own account.
-    """
+class Auth:
+    @staticmethod
+    def login_required():
+        pass
 
+    @property
+    def current_user(self):
```

Comment:
Questo e' un piccolo hack che commentiamo. Noi vogliamo incapsulare completamente in modo invisibile l'uso di flask-login.